### ISSUE TITLE
Some text is not visible in high contrast mode

### ISSUE THREAD

**Issue by Thomas1664** [microsoft/vscode-pull-request-github#4287]
To reproduce, change your theme to high contrast light.

The following UI is not visible in high contrast mode:

- code blocks in comments on the description view (also need a border in high contrast mode)
- code in comments is hard to distinguish from normal text
- text of the comment button at the end of the description view (Fixed by #4368)

**Comment by alexr00** [comment]
> code in comments is hard to distinguish from normal text

I didn't change this one as it's already using a VS Code color and it also has a different font.
